### DISCUSSION TITLE
[Feature] Create albums based on persons

### DISCUSSION THREAD

**Discussion by gbcatrinoiu** [immich-app/immich#6620] [Feature Request]
### The feature

Currently, albums can only be created by selecting photos. It would be very handy if creating albums based on a selection of people were possible, so that the album stays always updated with all the related photos of those individuals.

A use case would be sharing an album with a family member that includes all the photos of that specific person.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #1673

**Reply by gbcatrinoiu** [reply]
I tried looking for something similar but did not read the whole thread. Thanks for your comment!
